arlo-client/src/components/AuditForms/EstimateSampleSize.tsx

```diff
@@ -215,6 +215,18 @@ const EstimateSampleSize: React.FC<IProps> = ({
         setIsLoading(false)
         return
       }
+
+      const freezeResponse: IErrorResponse = await api(
```

Comment:
@MorganLove in the case of this single-page flow, `freeze` is called immediately. In the multi-jurisdiction flow, it would be called later in the flow.